xml/ha_config_cli.xml

```diff
@@ -94,25 +94,14 @@
   <tip>
    <title>Distinguish Between Shell Prompt and Interactive crm Prompt</title>
```

Comment:
I would change the title to 'Interactive crm Prompt' (because it is mostly about the interactive one now)

Reply:
Right!